frontend/awx/access/credentials/CredentialForm.tsx

```diff
@@ -578,7 +578,8 @@ function CredentialSubForm({
   setWatchedSubFormFields: (fields: unknown[]) => void;
 }) {
   const { t } = useTranslation();
-  const openCredentialPluginsModal = useCredentialPluginsModal();
+  const alertToaster = usePageAlertToaster();
```

Comment:
I would move this directly into `CredentialPluginsModal` and avoid passing it in to the `useCredentialPluginsModal ` as a hook.